index.html

```diff
@@ -4669,105 +4714,130 @@ <h3>Data First Approaches</h3>
       </p>
 
-      <ul>
-        <li>
-[[!JSON-LD]] supports the expression of language information using the
-<code>@language</code> feature, as described in [[!JSON-LD]] 1.1,
-<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">Section 4.2.3: String Internationalization</a>.
-        </li>
-        <li>
-[[!JSON-LD]] also supports the expression of language information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
-<a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
-This mechanism enables developers to use the internationalization features
-present in HTML to express language information. For example, consider the
-following key-value pair:
-
-          <pre>
-"nameHtml": "&lt;span lang="jp">和美&lt;/span>"
-          </pre>
-
-          <p>
-If the JSON-LD Context expressed that any value associated with the
-<code>nameHtml</code> property as being of type <code>rdf:HTML</code>, then
-software agents rendering the property can deterministically identify the
-language in use.
-          </p>
-        </li>
-      </ul>
-    </section>
+      <p>
+The general design pattern is to use the following markup template when
+expressing a text string that is tagged with a language and, optionally, a
+specific base direction:
+      </p>
 
-    <section class="informative">
-      <h3>Text direction</h3>
+      <pre class="example nohighlight" title="Design pattern for natural language strings">
+"<a>property</a>": {
+  "value": "<span class="highlight">The string value</span>",
+  "lang: "<code>LANGUAGE</code>"
+  "dir": "<code>DIRECTION</code>"
+}
+      </pre>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-to be able to explicitly encode the text direction. JSON and its derived formats
-use the UTF-8 encoding of [[Unicode]] for serializing and transmitting text.
-This enables text direction to be directly determined in simple cases,
-but the mechanism is insufficient for general use.
+An example utilizing the design pattern above is shown below to express the
+human-readable text "revoked" in the English language without specifying a
+text direction:
       </p>
 
+      <pre class="example nohighlight" title="Expressing natural language text as English">
+"statusReason": {
+  "value": "<span class="highlight">Revoked</span>",
+  "lang: "<code>en</code>"
+}
+      </pre>
+
       <p>
-[[!JSON-LD]] supports the expression of text layout information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
-<a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
-This mechanism enables developers to use the internationalization features
-present in HTML to express language information. For example, consider the
-following key-value pair:
-
-      <pre>
-"nameHtml": "&lt;span dir="rtl" lang="ar">HTML و CSS: تصميم و إنشاء مواقع الويب&lt;/span>"
```

Comment:
This text is a more meaningful example, because if you don't have the base direction it comes out wrong.

Reply:
Reverted to this example in 37daadbc5dc6d3e4a165c6e1393dd27d7eadf1ef.